views/contacts.ejs

```diff
@@ -5,28 +5,38 @@
 	<!-- Load bootstrap and fontawesome -->
 	<link rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.0.2/css/bootstrap.min.css">
 	<link rel="stylesheet" href="//netdna.bootstrapcdn.com/font-awesome/4.0.3/css/font-awesome.min.css">
+	<!-- css -->
+	<link rel="stylesheet" href="/styles.css">
 
-	<style>
-
-	</style>
 </head>
 <body>
```

Comment:
is there a way to template this so both .ejs files share their code outside of `<body>`?